app/assets/stylesheets/vendor/autoSuggest.css

```diff
@@ -1,212 +1,203 @@
-	border-radius: 5px;
+  margin: 2px 0 0 0;
+  padding: 0;
+  font-size: 13px;
+  color: #000;
+  background-color: #fff;
+  background-color: rgba(255,255,255,0.95);
+  z-index: 2;
+  box-shadow: 0 2px 12px #222;
+  border-radius: 5px;
 }
 
 li.as-result-item, li.as-message {
-	margin: 0 0 0 0;
-	padding: 5px;
-	background-color: transparent;
-	border: 1px solid #fff;
-	border-bottom: 1px solid #ddd;
-	cursor: pointer;
-	border-radius: 3px;
+  margin: 0 0 0 0;
+  padding: 5px;
+  background-color: transparent;
+  border: 1px solid #fff;
+  border-bottom: 1px solid #ddd;
+  cursor: pointer;
+  border-radius: 3px;
 }
 
 li:first-child.as-result-item {
-	margin: 0;
+  margin: 0;
 }
 
 li.as-message {
-	margin: 0;
-	cursor: default;
+  margin: 0;
+  cursor: default;
 }
 
 li.as-result-item.active {
-	background-color: #3668d9;
-	background-image: gradient(linear, 0% 0%, 0% 64%, from(rgb(110, 129, 245)), to(rgb(62, 82, 242)));
-	border-color: #3342e8;
-	color: #fff;
-	text-shadow: 0 1px 2px #122042;
+  background-color: #3668d9;
+  background-image: gradient(linear, 0% 0%, 0% 64%, from(rgb(110, 129, 245)), to(rgb(62, 82, 242)));
+  border-color: #3342e8;
+  color: #fff;
+  text-shadow: 0 1px 2px #122042;
 }
 
 li.as-result-item em {
-	font-style: normal;
-	background: #444;
-	padding: 0 2px;
-	color: #fff;
+  font-style: normal;
+  background: #444;
+  padding: 0 2px;
+  color: #fff;
 }
 
 li.as-result-item.active em {
-	background: #253f7a;
-	color: #fff;
+  background: #253f7a;
+  color: #fff;
 }
 
 /* Webkit Hacks  */
 @media screen and (-webkit-min-device-pixel-ratio:0) {
-	ul.as-selections li.as-selection-item {
-		padding-top: 6px;
-		padding-bottom: 6px;
-	}
-	ul.as-selections li.as-selection-item a.as-close {
-		margin-top: -1px;
-	}
-	ul.as-selections li.as-original input {
-		height: 20px;
-	}
+  ul.as-selections li.as-selection-item {
+    padding-top: 6px;
+    padding-bottom: 6px;
+  }
+  ul.as-selections li.as-selection-item a.as-close {
+    margin-top: -1px;
+  }
+  ul.as-selections li.as-original input {
+    height: 20px;
+  }
 }
 
 /* Opera Hacks  */
 @media all and (-webkit-min-device-pixel-ratio:10000), not all and (-webkit-min-device-pixel-ratio:0) {
-	ul.as-list {
-		border: 1px solid #ccc;
-	}
-	ul.as-selections li.as-selection-item a.as-close {
-		margin-left: 4px;
-		margin-top: 0;
-	}
+  ul.as-list {
+    border: 1px solid #ccc;
+  }
+  ul.as-selections li.as-selection-item a.as-close {
+    margin-left: 4px;
+    margin-top: 0;
+  }
 }
 
 /* IE Hacks  */
 ul.as-list {
-	border: 1px solid #ccc\9;
+  border: 1px solid #ccc\9;
 }
 ul.as-selections li.as-selection-item a.as-close {
-	margin-left: 4px\9;
-	margin-top: 0\9;
+  margin-left: 4px\9;
+  margin-top: 0\9;
 }
 
 /* Firefox 3.0 Hacks */
 ul.as-list,  x:-moz-any-link, x:default {
-	border: 1px solid #ccc;
+  border: 1px solid #ccc;
 }
 BODY:first-of-type ul.as-list, x:-moz-any-link, x:default { /* Target FF 3.5+ */
-	border: none;
+  border: none;
```

Comment:
I still wish this file wouldn't be further touched at all? No chance for overrides? This will only increase the pain when replacing it with an upstream version :/